packages/firestore/test/unit/specs/listen_spec.test.ts

```diff
@@ -570,34 +570,38 @@ describeSpec('Listens:', [], () => {
     );
   });
 
-  specTest('Omits global resume tokens for a short while', [], () => {
-    const query = Query.atPath(path('collection'));
-    const docA = doc('collection/a', 1000, { key: 'a' });
+  specTest(
+    'Omits global resume tokens for a short while',
+    ['no-memory-persistence'],
+    () => {
+      const query = Query.atPath(path('collection'));
+      const docA = doc('collection/a', 1000, { key: 'a' });
 
-    return (
-      spec()
-        .withGCEnabled(false)
-        .userListens(query)
-        .watchAcksFull(query, 1000, docA)
-        .expectEvents(query, { added: [docA] })
+      return (
+        spec()
+          .withGCEnabled(false)
+          .userListens(query)
+          .watchAcksFull(query, 1000, docA)
+          .expectEvents(query, { added: [docA] })
 
-        // One millisecond later, watch sends an updated resume token but the
-        // user doesn't manage to unlisten before restart.
-        .watchSnapshots(2000, [], 'resume-token-2000')
-        .restart()
+          // One millisecond later, watch sends an updated resume token but the
+          // user doesn't manage to unlisten before restart.
+          .watchSnapshots(2000, [], 'resume-token-2000')
+          .restart()
```

Comment:
Could we make restart check that the tag is set? Are there tests that have any business restarting without durable persistence?

Reply:
I left out this feature. We can add it if we need it.